docs/resources/Guild_Scheduled_Event.mdx

```diff
@@ -101,8 +102,164 @@ SCHEDULED --> CANCELED
+| by_month          | ?array of [recurrence rule - month](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-recurrence-rule-object-guild-scheduled-event-recurrence-rule-month)                          | Set of specific months to recur on                                                                                                               |
+| by_month_day      | ?array of int                                                                                                                                                                                | Set of specific dates within a month to recur on                                                                                                 |
+| by_year_day \[1\] | ?array of int                                                                                                                                                                                | Set of days within a year to recur on (1-364)                                                                                                    |
+| count  \[1\]      | ?int                                                                                                                                                                                         | The total amount of times that the event is allowed to recur before stopping                                                                     |
+
+\[1\] - Cannot be set externally currently.
+
+<Collapsible title="System limitations" description="Recurrence rules are powerful, but they have some specific restrictions" icon="warning">
+The current system limitations are present due to how reoccurring event data needs to be displayed in the client.
+In the future, we would like to open the system up to have fewer / none of these restrictions.
+
+###### The following fields cannot be set by the client / application
+  - `count`
+  - `end`
+  - `by_year_day`
+
+###### The following combinations are mutually exclusive
+  - `by_weekday`
+  - `by_n_weekday`
+  - `by_month` + `by_month_day`
+
+###### `by_weekday`
+  - Only valid for daily and weekly events (`frequency` of `DAILY` (`0`) or `WEEKLY` (`1`))
+  - when used in a daily event (`frequency` is `DAILY` (`0`))
+    - The values present in the `by_weekday` event must be a "known set" of weekdays.
+    - The following are current allowed "sets"
+      - Monday - Friday (`[0, 1, 2, 3, 4]`)
+      - Tuesday - Saturday (`[1, 2, 3, 4, 5]`)
+      - Sunday - Thursday (`[6, 0, 1, 2, 3]`)
+      - Friday & Saturday (`[4, 5]`)
+      - Saturday & Sunday (`[5, 6]`)
+      - Sunday & Monday (`[6, 0]`)
```

Comment:
one interesting thing we could note here is that we store the times in UTC. This means that a user in a different timezone may schedule an event for their Saturday/Sunday but this will actually be stored as Sunday/Monday. That's why those types of sets are allowed.